src/EnrollWebauthnController.js

```diff
@@ -88,9 +93,14 @@ function (Okta, Errors, FormType, FormController, CryptoUtil, webauthn, Footer,
               })
               .fail(function (error) {
                 self.trigger('errors:clear');
-                throw new Errors.WebAuthnError({
-                  xhr: {responseJSON: {errorSummary: error.message}}
-                });
+                // Do not display if it is abort error triggered by code when switching.
+                if (error && error.code === 20) {
```

Comment:
@gowthamidommety-okta @vincentvoong-okta  we can actually determine what the error message is in each case. I got it wrong previously as the console statements printed an object differently. FF as a string and chrome as object.

Reply:
Oops taking this back! Firefox triggers a different code vs chrome :/ /cc @mauriciocastillosilva-okta  trying to fix this now

Reply:
ah ok. Thanks for the update. Let me know what you find out, I'm curious about this

Reply:
in chrome the error code was 20 for programatic abort and 0 for user abort. But in firefox it is always 20. ( which is per spec) so I don't want to rely on this in consistent behavior.